tests/phpunit/tests/block-template.php

```diff
@@ -275,6 +281,35 @@ public function test_get_the_block_template_html_does_not_generally_enforce_loop
 		$this->assertSame( array( false, true ), $in_the_loop_logs, 'Main query loop was triggered incorrectly' );
 	}
 
+	/**
+	 * Tests that `get_the_block_template_html()` does not start the main query loop when on a template that is not from the current theme.
+	 *
+	 * @ticket 58154
```

Comment:
```suggestion
```
This new test method wasn't introduced or modified by ticket 58154, and doesn't test against a bug described by 58154.

From [Core Handbook - Writing PHPUnit Tests - Annotations](https://make.wordpress.org/core/handbook/testing/automated-testing/writing-phpunit-tests/#annotations):
> Use `@ticket 12345` to indicate that a test addresses the bug described in ticket `#12345`.

While I see the logic of including it because the test is historically related to the ticket and it's handy for grouping the tests, extrapolating that to a practice means a lot of tests would have a long list of `@ticket` annotations akin to `Follow-up to`, which isn't a standard practice, at least at this time.

There may be context I'm missing though, so let me know if that's the case.

Reply:
Happy to remove if that's the policy, but I am personally not sure that's the case. As you're saying, this has historically be a common practice and is helpful, even to see which ticket(s) this test is related to. I find having the ticket number here valuable, and removing it IMO wouldn't really improve anything.

Of course it could become a problem if a test ended up with dozens of ticket numbers, but realistically that hasn't been happening, so unless we adopt a clear policy such as "one ticket annotation per test" (since every bug is probably only clearly decribed in _one_ ticket), I think sticking to this existing practice makes sense.